src/components/Product/ListProduct.js

```diff
@@ -62,6 +52,38 @@ export default function ListProduct() {
     });
   };
 
+  const Delete = async (productid) => {
```

Comment:
Guys in this part I would have liked better if they used async and await approach if they use and it looked something like this also using try and catch to validate code

`const responseFirbease = await firebase
      .firestore()
      .collection(token)
      .doc(productid)
      .delete()`